lib/engine/game/base.rb

```diff
@@ -2061,7 +2061,7 @@ def next_sr_player_order
         self.class::NEXT_SR_PLAYER_ORDER
       end
 
-      def reorder_players(order = nil)
+      def reorder_players(order = nil, log_all_players: false)
```

Comment:
A bit of a nitpick, but would 'log_player_order' make more sense for the name of the kwarg?